app/views/thredded/posts_common/_post.html.erb

```diff
@@ -17,4 +17,8 @@
   <% if current_ability.can? :edit, post.original %>
     <%= link_to 'Edit Post', edit_post_path(post.original), class: 'thredded--post--edit' %>
   <% end %>
+
+  <% if current_ability.can? :destroy, post.original %>
+    <%= link_to 'Delete Post', messageboard_topic_post_path(post.messageboard.slug, post.postable.slug, post.original), method: :delete, class: 'thredded--post--delete' %>
```

Comment:
@glebm this helper `messageboard_topic_post_path` seems weird when just above there is `edit_post_path`. When using the regular `post_path` method the path that came out of it was the `topic-slug/page-1#post-1`. Is that intended behavior?